Sources/Leaf/Buffer/Buffer+Leaf.swift

```diff
@@ -126,12 +126,24 @@ extension Buffer {
     mutating func extractInstruction(stem: Stem) throws -> TagTemplate {
         let name = try extractInstructionName()
         let parameters = try extractInstructionParameters()
-
-        // check if body
-        moveForward()
-        guard current == .space, next == .leftCurlyBracket else {
+        
+        // check if body exists
+        
+        // Check for immediate curly brace without whitespace.
+        // If there is an immediate brace, do nothing.
+        if next != .leftCurlyBracket {
+            moveForward()
+        }
+        
+        // Move through any redundant whitespace
+        while current?.isWhitespace == true && next?.isWhitespace == true {
```

Comment:
I _believe_ there is a function on this object called `skipWhitespace()` if it exists, might be good to use that if behavior is same same.